fix linting issue in provider ui

falgunmpatel opened the pull request:
**Notes for Reviewers**

- This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

<!--
Thank you for contributing to Meshery! 

Contributing Conventions:

1. Include descriptive PR titles with [<component-name>] prepended.
2. Build and test your changes before submitting a PR. 
3. Sign your commits.
4. Include before and after screenshots/terminal output.

By following the community's contribution conventions upfront, the review process will 
be accelerated and your PR merged more quickly.
-->

falgunmpatel commented:
@leecalcote I’ve fixed the linting issues in the provider UI here.

Since my previous PR was for documentation, I created a separate PR for this fix. Would it be better to include such fixes in the original PR or keep them separate?

Thanks!

leecalcote commented:
Separate is ideal. 👍

leecalcote reviewed (changes requested):
Thank you for this, @falgunmpatel. Instead of fixing the linting errors in this way, will you please update the listing rule to be that of `key: value` with no whitespace after the `key`?

The current lint style is undesired.

falgunmpatel commented:
> Thank you for this, @falgunmpatel. Instead of fixing the linting errors in this way, will you please update the listing rule to be that of `key: value` with no whitespace after the `key`?
> 
> The current lint style is undesired.

Sure!

falgunmpatel commented:
Hey @Lee, I noticed that we aren’t using Prettier for the Provider UI. Is there a specific reason for that? If we were, we could align the ESLint configuration across both. If there are no plans to integrate Prettier, I can adjust some ESLint rules for now. Let me know your thoughts!"

falgunmpatel commented:
@leecalcote I’ve updated the ESLint configurations. I referenced the ESLint setup from the "UI" package for guidance, and it should be good now.

Let me know if you’d like any further updates! 🚀

leecalcote commented:
No, there’s no reason that we’re not using prettier here.

falgunmpatel commented:
@amitamrutiya
I need some help with fixing these failing E2E tests. Could you provide some guidance on how to resolve them?

falgunmpatel commented:
@amitamrutiya

I’ve fixed all ESLint issues and added consistency checks similar to meshery-ui. 

Let me know if any further refinements are needed. It's ok, if these are not worth merging yet.

leecalcote commented:
Good!
